projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -80,7 +80,14 @@ function wpcom_launchpad_get_task_definitions() {
 				add_action( 'publish_post', 'wpcom_launchpad_track_publish_first_post_task' );
 			},
 			'get_calypso_path'      => function ( $task, $default, $data ) {
-				return '/post/' . $data['site_slug_encoded'];
+				$base_path = '/post/' . $data['site_slug_encoded'];
+
+				// Add an answer_prompt query param for Write sites.
+				if ( 'write' === get_option( 'site_intent' ) ) {
+					return $base_path . '/?new_prompt=true';
```

Comment:
I think the extra `/` is unnecessary and gets taken out anyways, but does no harm.

Reply:
oh yes, maybe `addQueryArgs` would be fitting to add the query args.